### PR TITLE
Use judge secret in penalty screens

### PR DESCRIPTION
**Pull request by kostyanf14**
In general penalty screens are used by judges and judges can't check information with an upload secret.

Signed-off-by: Kostiantyn Kostiuk <kostyanf14@live.com>

### CODE REVIEW

File: lib/rails_admin/config/actions/device_ids_mismatch.rb [hunk 1]
```diff
@@ -15,8 +15,11 @@ class DeviceIdsMismatch < Base
             @collection = @object.solutions
               .where('solutions.device_id != users.device_id')
               .group('users.id')
-              .pluck('users.device_id', 'users.secret', 'array_agg(solutions.id)')
-              .map { |did, secret, solution_ids| [did, secret, Solution.includes(:task).where(id: solution_ids)] }
+              .pluck('users.device_id', 'users.judge_secret', 'array_agg(solutions.id)')
+              .map do |did, judge_secret, solution_ids|
+              [did, judge_secret,
+                Solution.includes(:task).where(id: solution_ids)]
```
Comment (just806me) [inline lib/rails_admin/config/actions/device_ids_mismatch.rb:21]: If you are going to split it to multiple lines, let's then use full name `device_id` for the first argument.

Also, is the indentation missing here?

File: lib/rails_admin/config/actions/ips_mismatch.rb [hunk 2]
```diff
@@ -15,8 +15,11 @@ class IpsMismatch < Base
             @collection = @object.solutions
               .where('solutions.ips != users.registration_ips')
               .group('users.id')
-              .pluck('users.registration_ips', 'users.secret', 'array_agg(solutions.id)')
-              .map { |ips, secret, solution_ids| [ips, secret, Solution.includes(:task).where(id: solution_ids)] }
+              .pluck('users.registration_ips', 'users.judge_secret', 'array_agg(solutions.id)')
+              .map do |ips, judge_secret, solution_ids|
+              [ips, judge_secret,
+                Solution.includes(:task).where(id: solution_ids)]
```
Comment (just806me) [inline lib/rails_admin/config/actions/ips_mismatch.rb:21]: If you are going to split it to multiple lines, let's then use full name `registration_ips` for the first argument.

Also, is the indentation missing here?

**Review by just806me — CHANGES_REQUESTED** [review]

File: lib/rails_admin/config/actions/device_ids_mismatch.rb (hunk 1 above)
Comment (kostyanf14) [reply lib/rails_admin/config/actions/device_ids_mismatch.rb:21]: Split and indentation were done by RuboCop. "did" will rename.

Comment (kostyanf14) [reply lib/rails_admin/config/actions/device_ids_mismatch.rb]: done

**Review by just806me — APPROVED** [review]

**Pull request merged**